solr/solr-ref-guide/src/other-parsers.adoc

```diff
@@ -24,7 +24,7 @@ Many of these parsers are expressed the same way as <<local-parameters-in-querie
 
 == Block Join Query Parsers
 
-There are two query parsers that support block joins. These parsers allow indexing and searching for relational content that has been <<nested-documents.adoc#nested-documents, indexed as Nested Documents>>.
+There are two query parsers that support block joins. These parsers allow indexing and searching for relational content that has been <<searching-nested-documents.adoc#searching-nested-documents, indexed as Nested Documents>>.
```

Comment:
I think we should link to the indexed side page because we're talking about that.

Reply:
Done